libcontainer/nsenter/nsexec.c

```diff
@@ -747,6 +758,17 @@ void receive_idmapsources(int sockfd)
 	receive_fd_sources(sockfd, "_LIBCONTAINER_IDMAP_FDS");
 }
 
+static void update_timens(char *map, size_t map_len)
+{
+	if (map == NULL || map_len == 0)
+		return;
+	write_log(DEBUG, "update /proc/self/timens_offsets to '%s'", map);
+	if (write_file(map, map_len, "/proc/self/timens_offsets") < 0) {
+		if (errno != EPERM)
```

Comment:
`EPERM` means `The caller does not have the CAP_SYS_TIME capability.`
Why ignore this error?

Reply:
Good catch. I think this was mistakenly copied from the `/proc/self/uid_map` and `gid_map` helpers (which call the setuid `newuidmap`/`newgidmap` helper programs in case of `-EPERM`).